sdk/mediaservices/Microsoft.Azure.Management.Media/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Microsoft.Azure.Management.Media release notes
 
+## Changes in 6.1.0
```

Comment:
The track 1 is being deprecated soon, let's talk why we need another new track 1 release now